mftf/2.3/getting-started.md

```diff
@@ -204,35 +198,110 @@ vendor/bin/mftf run:test AdminLoginTest --remove
 
 See more commands in [`mftf`][].
 
-### Step 9. Generate reports    {#reports}
+### Step 7. Generate reports    {#reports}
 
 During testing, the MFTF generates test reports in CLI.
 You can generate visual representations of the report data using [Allure Framework][].
 To view the reports in GUI:
 
--  [install Allure][]
--  run the tool to serve the artifacts in `dev/tests/acceptance/tests/_output/allure-results/`:
+- [install Allure][]
+- run the tool to serve the artifacts in `dev/tests/acceptance/tests/_output/allure-results/`:
```

Comment:
May want to capitalize the first words in the bullets.

Reply:
:ok: